web/urls.py

```diff
@@ -16,5 +16,7 @@
 	url(r'^edituser/$', views.edituser, name='pages-edituser'),
 	url(r'^privacy/$', views.privacy, name='pages-privacy'),
 	url(r'^project/(?P<project_id>[0-9]+)/$', views.project_detail),
```

Comment:
What is going on here? Why are some URLS named and others  aren't? That is some sloppy work!